jbt-it/neomdb
# #284 server retrieve list of trainees joined their internal project

How to test: 
Use Thunder Client to

1. Login with a user who has permission 14, add permission to member if necessary.
2. Run GET `Request https://localhost:3030/trainees/generations/:generationID/internal-projects-and-workshop-feedback` the result should include an array of 2 arrays with the requested data. If necessary, add some test data to the database.
